RPS.html

```diff
@@ -67,27 +67,61 @@
       }
     }
 
+    let playerScore = 0;
+    let computerScore = 0;
+
     function game(playerSelection) {
       const computerSelection = computerPlay();
 
       const resultGameDiv = document.querySelector('.result');
-      const pRoundRPS = document.createElement('p');
-      const pUserSelect = document.createElement('p');
-      const pCompSelect = document.createElement('p');
+      const roundResultParagraph = document.createElement('p');
+      const userChoiceParagraph = document.createElement('p');
+      const computerChoiceParagraph = document.createElement('p');
+      const playerScoreParagraph = document.createElement('p');
+      const computerScoreParagraph = document.createElement('p');
+      
+      const playerRockWin = "You won! Rock beats Scissors.";
+      const playerRockLose = "You lose! Rock beats Scissors.";
+      const playerPaperWin = "You won! Paper beats rock.";
+      const playerPaperLose = "You lose! Paper beats rock.";
+      const playerScissorsWin = "You won! Scissors beats paper.";
+      const playerScissorsLose = "You lose! Scissors beats paper";
```

Comment:
Why did you duplicate those? Those constants must be taken from a single place throughout the script at all times.